docs/_templates/autosummary/class_no_inherited_members.rst

```diff
@@ -0,0 +1,33 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+
+   {% block methods %}
+   .. automethod:: __init__
+
+   {% if methods %}
+   .. rubric:: {{ _('Methods') }}
+
+   .. autosummary::
+   {% for item in methods %}
+   {%- if item not in inherited_members %}
```

Comment:
This line, line 28, and their associated if-close statements are the only things added to this file from the [default class.rst](https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/autosummary/templates/autosummary/class.rst) in Sphinx.

Reply:
Resolving. Left a comment saying same thing in the file